src/index.js

```diff
@@ -41,15 +39,24 @@ class NetlifyAPI {
   }
 
   get accessToken() {
-    return (get(this, 'defaultHeaders.Authorization') || '').replace('Bearer ', '') || null
```

Comment:
`this.defaultHeaders` is set by the constuctors, so we can assume it is defined.